setup.cfg

```diff
@@ -1,5 +1,21 @@
 [flake8]
-exclude = build,.git,build-pypy,.tox
+exclude =
+    .git,
+    __pycache__,
+    docs/source/conf.py,
+    docs/conf.py,
+    old,
```

Comment:
This directory doesn't exist in dulwich

Reply:
Did you mean the `docs/source/conf.py` directory?

Reply:
I think it is possible that some of the contents of this project are not structured according to the directory structure of most projects for technical manuals, but I am not sure about this, but most of the `conf.py` I found will be in `docs/ Under the source/` directory.

So I want to ask why is it different?

Reply:
I didn't realize it is nonstandard. If this is causing problems somewhere, we can certainly talk about that.

Reply:
No problem, I am also studying, thank you for your feedback.